nautobot/extras/templates/extras/inc/jobresult_js.html

```diff
@@ -2,7 +2,7 @@
 {% load static %}
 
 <script type="text/javascript">
-{% if not result.date_done %}
+{% if result.status != 'FAILURE' and result.status != 'REVOKED' and result.status != 'SUCCESS' %}
```

Comment:
??
```suggestion
{% if result.status not in ['FAILURE', 'REVOKED', 'SUCCESS' %}
```

Reply:
You could also use [JobResultStatusChoices.READY_STATES](https://github.com/nautobot/nautobot/blob/8738d0b2d666202527728d10267059ce17c6fd32/nautobot/extras/choices.py#L203)

Reply:
How would I reference that choices value from a template? Maybe just my brain not working at the moment. :-)

Reply:
![image](https://github.com/nautobot/nautobot/assets/5603551/d6029231-43b6-408c-98da-3225fc093247)

Reply:
Django templates don't support that syntax. You could keep your current syntax or pass a template variable for `JobResultStatusChoices.READY_STATES` from the view.